.circleci/config.yml

```diff
@@ -19,54 +25,55 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: vendor-{{ checksum "Gopkg.lock" }}
+          key: go-mod-v1-{{ checksum "go.sum" }}
```

Comment:
I reran the workflow on CircleCI, and it said:
> No cache is found for key: go-mod-v1-5m8iTjU8woo3ixYf_mEKZ4D_98q5g9IaxeuniYq+mTU=

Is the cache working right?

Reply:
It should be. This is actualy how circle recommend configuring their cache for go modules.

https://circleci.com/blog/go-v1.11-modules-and-circleci/#caching

I can poke and prod a bit more. Perhaps our cache got invalidated for some reason.